client/src/use/useGirderDataset.ts

```diff
@@ -20,7 +20,7 @@ interface VIIMEDataset extends GirderModel {
 
 export default function useGirderDataset() {
   const dataset = ref(null as VIIMEDataset | null);
-  const imageUrls = ref([] as string[]);
+  const imageUrls = ref([] as Record<string, any>);
```

Comment:
Rename the variable to `imageData` just to indicate that it contains more than just the URLs.  This will probably need to be updated elsewhere.  Also we can create an interface for this type or change it to `Record<string, string>` because that is technically the object format for the `imageData`